### PR TITLE
TiCDC: Update manage-ticdc.md

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

Update manage-ticdc.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3141, https://github.com/pingcap/docs-cn/pull/3352, https://github.com/pingcap/docs-cn/pull/2909
- Other reference link(s):

### CODE REVIEW

**Comment by yikeke** [comment]
Please help review this PR. @ran-huang

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -7,11 +7,47 @@ aliases: ['/docs/dev/reference/tools/ticdc/manage/']
 
 # Manage TiCDC Cluster and Replication Tasks
 
-Currently, you can manage a TiCDC cluster and replication tasks using the `cdc cli` command-line tool or the HTTP interface.
+This document describes how to deploy a TiCDC cluster and how to manage the TiCDC cluster and replication tasks through the command line tool `cdc cli` and the HTTP interface.
+
+## Deploy TiCDC
+
+### Using TiUP to deploy TiCDC
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:14]:
```suggestion
### Use TiUP
```

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -7,11 +7,47 @@ aliases: ['/docs/dev/reference/tools/ticdc/manage/']
...
+
+```shell
+cdc server --pd=http://10.0.10.25:2379 --log-file=ticdc_1.log --addr=0.0.0.0:8301 --advertise-addr=127.0.0.1:8301
+cdc server --pd=http://10.0.10.25:2379 --log-file=ticdc_2.log --addr=0.0.0.0:8302 --advertise-addr=127.0.0.1:8302
+cdc server --pd=http://10.0.10.25:2379 --log-file=ticdc_3.log --addr=0.0.0.0:8303 --advertise-addr=127.0.0.1:8303
+```
+
+The following are descriptions of options available in the `cdc server` command:
+
+- `gc-ttl`: The TTL (Time To Live) of the service level `GC safepoint` in PD set by TiCDC, specified in seconds. The default value is `86400`, which means 24 hours.
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:40]:
```suggestion
- `gc-ttl`: The TTL (Time To Live) of the service level `GC safepoint` in PD set by TiCDC, in seconds. The default value is `86400`, which means 24 hours.
```

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -7,11 +7,47 @@ aliases: ['/docs/dev/reference/tools/ticdc/manage/']
...
+- `pd`: URL of the PD client.
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:41]:
```suggestion
- `pd`: The URL of the PD client.
```

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -7,11 +7,47 @@ aliases: ['/docs/dev/reference/tools/ticdc/manage/']
...
+- `addr`: The listening address of TiCDC, the HTTP API address, and the Prometheus address of the service.
+- `advertise-addr`: The access address of TiCDC to the outside world.
+- `tz`: Time zone used by the TiCDC service. TiCDC uses this time zone when time data types such as `TIMESTAMP` are converted internally or when data are replicated to the downstream. The default is the local time zone in which the process runs.
+- `log-file`: The address of the running log of the TiCDC process. The default is `cdc.log`.
+- `log-level`: The log level when the TiCDC process is running. The default is `info`.
 
 ## Use `cdc cli` to manage cluster status and data replication task
 
-This section introduces how to use `cdc cli` to manage a TiCDC cluster and data replication tasks.
+This section introduces how to use `cdc cli` to manage a TiCDC cluster and data replication tasks. In the following interface description, it is assumed that PD listens on `127.0.0.1` and the port is `2379`.
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:50]:
```suggestion
This section introduces how to use `cdc cli` to manage a TiCDC cluster and data replication tasks. The following interface description assumes that PD listens on `127.0.0.1` and the port is `2379`.
```

File: ticdc/manage-ticdc.md [hunk 5]
```diff
@@ -38,61 +74,172 @@ This section introduces how to use `cdc cli` to manage a TiCDC cluster and data
...
+}
+```
+
+In the above command:
+
+- `resolved-ts`: The largest transaction `TS` in the current `changfeed`. Note that this `TS` has been successfully sent from TiKV to TiCDC.
+- `checkpoint-ts`: The largest transaction `TS` in the current `changefeed` that has been successfully written to the downstream.
+- `admin-job-type`: The status of a `changefeed`:
+    - `0`: The state is normal. It is the initial status.
+    - `1`: The task is paused. When the task is paused, all replicated `processor`s exit, while the configuration and the replication status of the task are retained.
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:198]:
```suggestion
    - `1`: The task is paused. When the task is paused, all replicated `processor`s exit. The configuration and the replication status of the task are retained, so you can resume the task from `checkpiont-ts`.
```

File: ticdc/manage-ticdc.md [hunk 6]
```diff
@@ -196,70 +343,6 @@ For nodes other than owner nodes, executing the above command will return the fo
...
-{
- "status": true,
- "message": ""
-}
-```
-
-- `admin-job=3` means to delete the replication task. After the TiCDC server receives the request, all `processor`s are stopped and the configuration information of the task is cleared. The replication status is reserved. No service is available except for the query.
-- `cf-id=xxx` is the ID of `changefeed` that needs operation.
-
 ## Error handling
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:346]: I don't see this section in the Chinese doc: https://docs.pingcap.com/zh/tidb/dev/manage-ticdc.
Is it deleted?

File: ticdc/manage-ticdc.md [hunk 7]
```diff
@@ -38,61 +74,172 @@ This section introduces how to use `cdc cli` to manage a TiCDC cluster and data
...
+                "resolved-ts": 415241860902289409,
+                "checkpoint-ts": 415241860640145409,
+                "admin-job-type": 0
+        }
+}
+```
+
+In the above command:
+
+- `resolved-ts`: The largest transaction `TS` in the current `changfeed`. Note that this `TS` has been successfully sent from TiKV to TiCDC.
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:194]: Please also update the Chinese doc.
```suggestion
- `resolved-ts`: The largest transaction `TS` in the current `changefeed`. Note that this `TS` has been successfully sent from TiKV to TiCDC.
```

**Comment by ran-huang** [comment]
@toutdesuite Made a commit. PTAL~

**Comment by toutdesuite** [comment]
> @toutdesuite Made a commit. PTAL~

LGTM, thanks

File: ticdc/manage-ticdc.md (hunk 6 above)
Comment (toutdesuite) [reply ticdc/manage-ticdc.md]: removed in 26d18ec595a4890ed699b0f1f838f7b5f78ceb9c

**Pull request merged**